openprescribing/dmd2/tests/test_import_dmd2.py

```diff
@@ -147,6 +147,21 @@ def test_another_import(self):
         # that the VMP with VPID 22480211000001104 has been updated with a new
         # VPID (12345).  This VMP now has a VPIDPREV field, and all references
         # to the old VPID have been updated to the new VPID.
+        #
+        # Additionally, there is now an NCSOConcession with a FK reference to
+        # an existing VMPP.
+
+        vmpp = VMPP.objects.get(id=22479511000001101)
+        concession = NCSOConcession.objects.create(
+            date='2019-06-01',
+            vmpp=vmpp,
+            drug=vmpp.nm,
+            pack_size=vmpp.qtyval,
+            price_pence=123
+        )
+
+        vmpp.delete()
```

Comment:
Comment here to explain why we're doing it? And/or assertion that concession still exists?

Reply:
Oops, that shouldn't be there.